### PR TITLE
[DEV-622] Remove crawler block

### PR DESCRIPTION
**Pull request by tommaso1**
#### List of Changes

Removed robots.txt file

#### Motivation and Context
Now web robots (typically search engine robots) can crawl pages on this website. 

#### How Has This Been Tested?

Can be tested adding `.env.local` new ENV variables ALLOW_CRAWLER="true" and run `npm run build -w nextjs-website`. In out folder should generate `robots.txt` file like this:
```
User-Agent: *
Allow: /

``` 

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/public/robots.txt
Comment (datalek) [inline apps/nextjs-website/public/robots.txt:1]: By removing this file, both environments can be scanned, but the ticket says that only the production environment should be. As you described on PR's body we should use `robots.ts`

**Review by datalek — CHANGES_REQUESTED** [review]

**Comment by kin0992** [comment]
We need to wait to merge this PR

**Review by marcobottaro — APPROVED** [review]

File: .github/workflows/code_review.yaml [hunk 1]
```diff
@@ -78,6 +78,7 @@ jobs:
         with:
           path_to_gitbook_docs: ${{ github.workspace }}/apps/nextjs-website/docs
           cookie_domain_script: ${{ secrets.COOKIE_DOMAIN_SCRIPT }}
+          allow_crawler: ${{ secrets.ALLOW_CRAWLER }}
```
Comment (kin0992) [inline .github/workflows/code_review.yaml:81]:
```suggestion
          allow_crawler: ${{ vars.ALLOW_CRAWLER }}
```

File: .github/workflows/deploy_website.yaml [hunk 2]
```diff
@@ -45,6 +45,7 @@ jobs:
           environment: ${{ matrix.environment }}
           path_to_gitbook_docs: ${{ github.workspace }}/apps/nextjs-website/docs
           cookie_domain_script: ${{ secrets.COOKIE_DOMAIN_SCRIPT }}
+          allow_crawler: ${{ secrets.ALLOW_CRAWLER }}
```
Comment (kin0992) [inline .github/workflows/deploy_website.yaml:48]:
```suggestion
          allow_crawler: ${{ vars.ALLOW_CRAWLER }}
```

File: .github/workflows/deploy_website.yaml [hunk 3]
```diff
@@ -67,6 +68,7 @@ jobs:
           environment: ${{ inputs.environment }}
           path_to_gitbook_docs: ${{ github.workspace }}/apps/nextjs-website/docs
           cookie_domain_script: ${{ secrets.COOKIE_DOMAIN_SCRIPT }}
+          allow_crawler: ${{ secrets.ALLOW_CRAWLER }}
```
Comment (kin0992) [inline .github/workflows/deploy_website.yaml:71]:
```suggestion
          allow_crawler: ${{ vars.ALLOW_CRAWLER }}
```

**Review by kin0992 — CHANGES_REQUESTED** [review]

**Review by kin0992 — APPROVED** [review]

**Review by datalek — APPROVED** [review]

**Pull request merged**